tests/engine/test_flow_runner.py

```diff
@@ -1540,7 +1540,7 @@ def log_stuff():
     assert flow_state.result[log_stuff].result == (42, "original")
 
     time.sleep(0.75)
-    assert len(calls) == 1
+    assert len(calls) >= 1
     assert len(calls[0][0]) == 6  # actual number of logs
```

Comment:
This part also is nondeterministic and is failing on circle rn, sounds like we expect all 6 to be spread out amongst all of the calls first arg... so maybe safer to sum([call[0] for call in calls]) or something to get the 6 logs

Reply:
Updated